src/annotator/plugin/toolbar.coffee

```diff
@@ -27,6 +30,10 @@ module.exports = class Toolbar extends Plugin
     else
       $(@element).append @toolbar
 
+    config = configFrom(window);
```

Comment:
old PR for reference

https://github.com/hypothesis/client/commit/d7acf2f3e0bfe5d5f38d1c5491c2ea260b903fbb

`This is generally the new part of this diff. Instead of looking into `window.hypothesisConfig` which in most cases is not even there, it now calls `configFrom`. This does mean that configFrom is called twice now, once from `annotator/index.js` and then again here, but the hope is that this is a better source of truth, but I'm still not sure this is correct.